packages/client/hmi-client/src/components/widgets/tera-infinite-canvas.vue

```diff
@@ -164,8 +164,6 @@ main > * {
 
 .background-layer {
 	cursor: grab;
-	width: 100%;
-	height: 100%;
```

Comment:
Why I got rid of these:

The 100%s were shifting the whole project component up when opening the new panel. They are redundant anyways since the width and height of the svg is assigned in the HTML